### PR TITLE
Update engagement endpoint callback info

### PR DESCRIPTION
**Pull request by tonylepmets**
JSDK-6

Can be merged after this is deployed: https://github.com/salemove/backend-web/pull/2045

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -230,11 +230,11 @@ $.ajax({
 
 #### ENGAGEMENT_START
 
-Triggered when the visitor engages or is engaged by an operator
+Triggered when the visitor engages or is engaged by an operator, callback returns engagement ID.
```
Comment (indrekj) [inline apiary.apib]: not really, it returns an object which has engagement id. not sure about the wording though

**Comment by tonylepmets** [comment]
Not sure either about wording, something like this perhaps

File: apiary.apib [hunk 2]
```diff
@@ -230,11 +230,26 @@ $.ajax({
...
 
-Triggered when the visitor engages or is engaged by an operator
+Triggered when the visitor engages or is engaged by an operator.
+
++ Response - engagement information
+
+```javascript
+{
+  engagementId: 608
+}
```
Comment (wifilinker) [inline apiary.apib]: what about ending the ```

File: apiary.apib [hunk 3]
```diff
@@ -230,11 +230,26 @@ $.ajax({
...
-Triggered when an engagement the visitor was in has ended
+Triggered when an engagement the visitor was in has ended.
+
++ Response - engagement information
+
+```javascript
+{
+  engagementId: 699
+}
+
```
Comment (wifilinker) [inline apiary.apib]: and here

**Comment by tonylepmets** [comment]
Updated

**Comment by deiwin** [comment]
It can be merged when salemove/backend-web#2045 is merged. No need to wait for deploy. We have a release branch in this repo that gets published to the docs website. Stuff in development can be merged into master without any issues.

**Comment by deiwin** [comment]
!squash

**Comment by deiwin** [comment]
I will give the JS SDK part of the docs an overhaul soon. I think this is okay until then as its more or less in line with the rest that's currently in there

**Pull request merged**